### PR TITLE
FIX Scrutinizer

### PR DESCRIPTION
**Pull request by inoveaconseil**

### CODE REVIEW

File: htdocs/compta/salaries/class/paymentsalary.class.php [hunk 1]
```diff
@@ -115,13 +116,13 @@ function update($user=null, $notrigger=0)
 		$error=0;
 
 		// Clean parameters
-		$this->fk_user=trim($this->fk_user);
+		$this->fk_user=$this->fk_user;
```
Comment (frederic34) [inline htdocs/compta/salaries/class/paymentsalary.class.php:119]: usually i replace trim by (int) to clean parameters
ex: https://github.com/Dolibarr/dolibarr/pull/9797/files

File: htdocs/ecm/class/ecmfiles.class.php [hunk 2]
```diff
@@ -99,8 +99,9 @@ class EcmFiles extends CommonObject
 	public $acl;
 	public $src_object_type;
 	public $src_object_id;
-
-
+        public $errors;
```
Comment (frederic34) [inline htdocs/ecm/class/ecmfiles.class.php:102]: $errors is in commonobject

File: htdocs/compta/salaries/class/paymentsalary.class.php (hunk 1 above)
Comment (eldy) [inline htdocs/compta/salaries/class/paymentsalary.class.php:119]:
```suggestion
		$this->fk_user=(int) $this->fk_user;
```

File: htdocs/ecm/class/ecmfiles.class.php (hunk 2 above)
Comment (eldy) [inline htdocs/ecm/class/ecmfiles.class.php:102]:
```suggestion
```

File: htdocs/ecm/class/ecmfiles.class.php [hunk 3]
```diff
@@ -165,10 +166,10 @@ public function create(User $user, $notrigger = false)
 			 $this->extraparams = trim($this->extraparams);
 		}
 		if (isset($this->fk_user_c)) {
-			 $this->fk_user_c = trim($this->fk_user_c);
+			 $this->fk_user_c = $this->fk_user_c;
```
Comment (eldy) [inline htdocs/ecm/class/ecmfiles.class.php:169]: $this->fk_user_c = $this->fk_user_c;
Are you sure this is equal ;-)

File: htdocs/ecm/class/ecmfiles.class.php [hunk 4]
```diff
@@ -165,10 +166,10 @@ public function create(User $user, $notrigger = false)
...
 		}
 		if (isset($this->fk_user_m)) {
-			 $this->fk_user_m = trim($this->fk_user_m);
+			 $this->fk_user_m = $this->fk_user_m;
```
Comment (eldy) [inline htdocs/ecm/class/ecmfiles.class.php:172]: New code seems useless.
May be you want to do 
$this->fk_user_m = (int) $this->fk_user_m
?

**Comment by eldy** [comment]
This PR is now too old, so i close it. A lot of code may have been validated since the PR and the PR may be obsolete, or too difficult to merge due to other changes. Best choice is too cancel it to avoid to keep PR with too low chances to be validated.

**Pull request closed without merging**